mopidy_ytmusic/backend.py

```diff
@@ -58,18 +57,12 @@ def __init__(self, config, audio):
         self.stream_preference = config["ytmusic"]["stream_preference"]
         self.verify_track_url = config["ytmusic"]["verify_track_url"]
 
-        if config["ytmusic"]["auth_json"]:
-            self._ytmusicapi_auth_json = config["ytmusic"]["auth_json"]
+        if "oauth_json" in config["ytmusic"]:
+            self.api = YTMusic(auth=config["ytmusic"]["oauth_json"])
+            self.auth = True
```

Comment:
This is just a comment - I don't have any issue with your code.

When I was proposing my changes to use the newer authorisation method, I didn't want to overwrite user files, so I added a separate `oauth_json` section to the config and left `auth_json` intact. Later on it made me misread `self.auth` variable as "authorising through browser", I think, so I added a separate `self.oauth` for "authorising through API". Nowadays, I think the original intention for `self.auth` was to have a boolean equal to `True` whenever the app is authorised, regardless of how it was authorised. If I knew other devs will have to deal with my changes, I'd just overwrite the damn user file. :laughing: